samples/Mvc.Client/Startup.cs

```diff
@@ -17,10 +17,30 @@ public class Startup
     {
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddAuthentication(options =>
-            {
-                options.SignInScheme = CookieAuthenticationDefaults.AuthenticationScheme;
-            });
+            services.AddAuthentication(CookieAuthenticationDefaults.AuthenticationScheme)
+                .AddCookie(o => {
+                    o.LoginPath = new PathString("/login");
+                    o.LogoutPath = new PathString("/signout");
+                })
+                .AddOpenId<OpenIdAuthenticationOptions, OpenIdAuthenticationHandler<OpenIdAuthenticationOptions>>("Orange", "Orange", options =>
```

Comment:
Outch, my eyes are bleeding. Can't we just use the non-generic overload here? :sweat_smile: